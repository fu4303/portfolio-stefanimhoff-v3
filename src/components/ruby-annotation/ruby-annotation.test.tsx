import React from 'react';
import { axe } from 'jest-axe';

import { render } from '../../services/test-utils';

import { RubyAnnotation } from '.';

describe('RubyAnnotation', () => {
  test('renders correctly', () => {
    const { container } = render(<RubyAnnotation base="本" text="ほん" />);

    expect(container.firstChild).toMatchSnapshot();
  });

  test('is accessible', async () => {
    const { container } = render(<RubyAnnotation base="本" text="ほん" />);
    const results = await axe(container);

    expect(results).toHaveNoViolations();
  });
});
