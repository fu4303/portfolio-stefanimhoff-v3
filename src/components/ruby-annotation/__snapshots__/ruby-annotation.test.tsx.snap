// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RubyAnnotation renders correctly 1`] = `
.c0 {
  font-size: clamp(1.00rem,1.1vw,1.25rem);
}

<ruby
  class="c0"
>
  本
  <rp>
    （
  </rp>
  <rt>
    ほん
  </rt>
  <rp>
    ）
  </rp>
</ruby>
`;
